src/config.coffee

```diff
@@ -36,6 +36,40 @@ nconf.defaults
     params: []
   dataDir: SBS_HOME
 
+# Post-process custom files and params
+customFields = nconf.get('customFields')
+
+# Ensure array
+customFields.files = customFields.files || []
+# Always add upload_file_minidump file as first file
+customFields.files.splice(0, 0,
+  name: 'upload_file_minidump'
+  downloadAs: 'upload_file_minidump.#{id}.dmp'
```

Comment:
Handlebars seems a little overkill, but would work ^_^.  I needed this so that each download gets a unique name (saves an extra click and/or typing when going through dumps and opening them in the debugger - don't get the overwrite prompt each time, and you can't overwrite a dump that's actively being debugged anyway).  I didn't expect any other fields would want this, but thinking more, they might.  I'm considering throwing in a screenshot capture as one of my custom uploaded files, and it would be useful if I could configure that to save the screenshot right next to the dump for future reference, so might be generally useful.  With handlebars, I think we have to be explicit about what values can be used in the templates - it might be useful to have the other attributes (e.g. be able to add IP, or custom fields like an account name, or crash timestamp, through a template), but maybe just "id" for the express purpose of a unique filename is sufficient for now.